components/clickup/sources/new-task/new-task.mjs

```diff
@@ -0,0 +1,31 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "clickup-new-task",
+  name: "New Task (Instant)",
+  description: "Emit new event when a new task is created",
+  version: "0.0.1",
+  dedupe: "unique",
+  type: "source",
+  methods: {
+    ...common.methods,
+    _getMeta({ task_id: taskId }) {
+      return {
+        id: taskId,
+        summary: taskId,
```

Comment:
Again, giving a summary more than `taskId` might be better for user experience.

Reply:
The webhook payload don't provide more informations :(